examples/config/tests/development.rs

```diff
@@ -1,6 +1,7 @@
 #![feature(proc_macro_hygiene)]
 
-#[macro_use] extern crate rocket;
+#[macro_use]
+extern crate rocket;
```

Comment:
Is there any configuration that would keep these together on one line? Same with some `#[cfg]` lines.

Reply:
I guess so, let me dig into this one

Reply:
Looks like `inline_attribute_width` should do the trick
(Reference)[https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#inline_attribute_width]